doc/cla/individual/opa-cormaza.md

```diff
@@ -0,0 +1,11 @@
+Ecuador, 2021-08-20
+
+I hereby agree to the terms of the Odoo Individual Contributor License
```

Comment:
It seems your company has already signed the Corporate CLA, so unless you own the copyright to this code personally, your name should be added instead in the contributors for your organization:
https://github.com/odoo/odoo/blob/master/doc/cla/corporate/opa-consulting.md

The CLA implies that you are granting a license to all code you contribute, so you must be the copyright owner. In many companies and countries, employment contracts have a clause where the copyright of the works employees produce for the company (including code) belongs to the company. You may need to double-check that :)